bodhi/server/util.py

```diff
@@ -343,6 +343,20 @@ def status2html(context, status):
     return "<span class='label label-%s'>%s</span>" % (cls, status)
 
 
+def ci_status2html(context, status):
+    cls = {
+        'ignored': 'success',
+        'running': 'warning',
+        'passed': 'success',
+        'failed': 'danger',
+        'queued': 'info',
+        'unknown': 'primary',
+    }[status]
+    if status == 'unknown':
+        status = 'info awaited'
```

Comment:
Since a great many existing updates are going to have ```None``` as their ```Build.ci_status```, I think this will cause them to show "Tests info awaited". I think we instead probably just don't want to display CI Status for historic builds at all.